DESCRIPTION

```diff
@@ -33,6 +33,7 @@ Imports:
     stringr (>= 1.5.0),
     svglite (>= 2.1.0),
     sysfonts (>= 0.8.8)
+Remotes: JackBuckley/e61-chart-maker

```

Comment:
I can't accept this commit. It will result in the same problem as what was resolved by this commit: https://github.com/e61-Institute/theme61/commit/cb1887981944281b0daed789153d272b004d909f

Reply:
What is the problem that it causes?

Reply:
Github Actions fails to build the package and the package website because the virtual machine it creates cannot access the thing in Remotes for an unknown reason.